### ISSUE TITLE
requirements-new.txt unusable

### ISSUE THREAD

**Issue by erc7** [KutsuyaYuki/DeepCreamPy#2]
`requirements-new.txt` from the latest push references multiple files on local, e.g.

https://github.com/KutsuyaYuki/DeepCreamPy/blob/d07cb8d1767485054c68e0adad3cc965d5882430/requirements-new.txt#L6

I'm assuming this was supposed to be the requirements file that was supposed to be used with the compatibility fix for newer CUDA versions from the latest commit to master?

**Comment by KutsuyaYuki** [comment]
Yes, this was a very early test to get it working with a quickly generated requirements file.
